### DISCUSSION TITLE
Print  image on Air for Android

### DISCUSSION THREAD

**Discussion by JoseEstevesPt** [airsdk/Adobe-Runtime-Support#794] [Q&A]
I just looked everywhere and could not find how to print from an Android Air app, on desktop you can use PrintJob(), on iOS there is an ANE (https://github.com/janumedia/AirPrintANE/). Could not find info for Android on printing or using the print service... can someone help?

**Comment by marchbold** [comment]
You can contact us if you need something developed (airnativeextensions@distriqt.com). We haven't done an Android printing extension as yet, but it should be possible from a quick look over the docs.
